translatorbot/daily-bot-manager.py

```diff
@@ -0,0 +1,51 @@
+import os
+import requests
+import subprocess
+import time
+
+from flask import Flask, jsonify
+from flask_cors import CORS
+from auth import get_meeting_token
+
+app = Flask(__name__)
+CORS(app)
+
+@app.route('/spin-up-bot', methods=['POST'])
+def spin_up_bot():
+    daily_api_key = os.getenv('DAILY_API_KEY')
+    api_path = os.getenv('DAILY_API_PATH') or 'https://api.daily.co/v1'
+
+    timeout = int(os.getenv("BOT_MAX_DURATION") or 300)
+    exp = time.time() + timeout
+    res = requests.post(f'{api_path}/rooms',
+                        headers={'Authorization': f'Bearer {daily_api_key}'},
+                        json={
+                            'properties': {
+                                'exp': exp,
+                                'enable_chat':True,
+                                'enable_emoji_reactions': True,
+                                'eject_at_room_exp': True,
+                                'enable_prejoin_ui': False,
+                                'enable_recording': 'cloud'
+                            }
+                        })
+    if res.status_code != 200:
+        return jsonify({'error': 'Unable to create room', 'status_code': res.status_code, 'text': res.text}), 500
+    room_url = res.json()['url']
```

Comment:
Suggest getting `res.json()` once and assigning to a var to access `'url'` and `'name'` (instead of reinvoking twice here)